lib/collection/src/shard/forward_proxy_shard.rs

```diff
@@ -97,9 +100,21 @@ impl ForwardProxyShard {
         let points: Result<Vec<PointStruct>, String> =
             batch.into_iter().map(|point| point.try_into()).collect();
 
-        let insert_points_operation = CollectionUpdateOperations::PointOperation(
-            PointOperations::UpsertPoints(PointInsertOperations::PointsList(points?)),
-        );
+        let points = points?;
+
+        let insert_points_operation = if sync {
+            CollectionUpdateOperations::PointOperation(PointOperations::SyncPoints(
+                PointSyncOperation {
+                    from_id: points.first().map(|p| p.id),
+                    to_id: points.last().map(|p| p.id),
```

Comment:
First batch is supposed to start with `from_id: None` and the last one with `to_id: None`